scripts/populate_preprint_providers.py

```diff
@@ -360,14 +361,666 @@ def main():
                 (['Law'], True),
                 (['Social and Behavioral Sciences'], True),
             ],
-        },
+        }
     ]
 
+    if env == 'stage':
+        STAGING_PREPRINT_PROVIDERS = [
```

Comment:
The only problem is that some providers require the ` use_plos ` flag to be set to properly create their attribute dict. So those provider dictionaries need to be created at runtime.